src/javascript/app/Stores/Modules/Trading/Helpers/symbol.js

```diff
@@ -1,4 +1,25 @@
+import { LocalStore } from '_common/storage';
+
 export const pickDefaultSymbol = (active_symbols = []) => {
     if (!active_symbols.length) return '';
+    return getFavoriteOpenSymbol(active_symbols) || getFirstOpenSymbol(active_symbols);
+};
+
+const getFavoriteOpenSymbol = (active_symbols) => {
+    if (LocalStore.get('cq-favorites') === undefined) return undefined;
+    const client_favorite_symbols = JSON.parse(LocalStore.get('cq-favorites'))['chartTitle&Comparison'];
```

Comment:
```suggestion
    const client_favorite_symbols = JSON.parse(chart_favorites)['chartTitle&Comparison'];
```

Reply:
This one is related to assign `LocalStore` to a variable,
Sure I'll change this too.